src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2_img2img.py

```diff
@@ -367,11 +367,16 @@ def __call__(
             if XLA_AVAILABLE:
                 xm.mark_step()
 
+        if output_type not in ["pt", "np", "pil", "latent"]:
+            raise ValueError(
+                f"Only the output types `pt`, `pil`, `np` and `latent` are supported not output_type={output_type}"
+            )
```

Comment:
This can be removed. Refer to other pipelines for an example.

https://github.com/huggingface/diffusers/blob/31c4f24fc18b7b451bd186e2132fa8bb3610374a/src/diffusers/pipelines/flux/pipeline_flux.py#L974-L981